modules/insights/private-link-scope/main.bicep

```diff
@@ -6,6 +6,14 @@ metadata owner = 'Azure/module-maintainers'
 @minLength(1)
 param name string
 
+@description('''
+  Required. Specifies the access mode of ingestion or queries through associated private endpoints in scope. For security reasons, it is recommended to use PrivateOnly whenever possible to avoid data exfiltration.
```

Comment:
Shouldn't this be set to Optional?

Reply:
Never mind this comment. I just validated that latest API sets this as a required parameter

Reply:
Yes, correct. Fixed!